Lib/test/test_genericalias.py

```diff
@@ -95,6 +95,17 @@ def test_exposed_type(self):
         self.assertIs(a.__origin__, list)
         self.assertEqual(a.__parameters__, (int,))
 
+    def test_isinstance(self):
+        self.assertTrue(isinstance([], list))
+        with self.assertRaises(TypeError):
+            isinstance([], list[str])
+
+    def test_issubclass(self):
+        class T(list): ...
+        self.assertTrue(issubclass(T, list))
+        with self.assertRaises(TypeError):
+            issubclass(T, list[str])
```

Comment:
I'd rename the class to `L` -- in my head, `T` is strongly connected to type variables.